app/views/layouts/application.html.haml

```diff
@@ -3,7 +3,13 @@
   %head
     %meta{charset: 'utf-8'}
     %meta{content: 'width=device-width, initial-scale=1.0', name: 'viewport'}
+
+    %meta{name: 'title', content: 'hackEDU'}
+    %meta{name: 'description', content: 'The international movement of free student-led coding clubs.'}
+    %meta{name: 'og:image', content: "#{ image_path('logo.png') }"}
+
     %title= full_title(yield(:title))
+
```

Comment:
Nope, totally arbitrary. Seemed easier to read having the meta tags separated.

Perhaps a better solution would be to move the meta tags to a partial and not have the extra newlines?